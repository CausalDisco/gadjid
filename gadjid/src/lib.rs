// SPDX-License-Identifier: MPL-2.0
#![warn(missing_docs)]
//! gadjid -  Graph Adjustment Identification Distance library
mod ascending_list_utils;
mod graph_loading;
pub mod graph_operations;
mod partially_directed_acyclic_graph;

pub use graph_loading::constructor::EdgelistIterator;
pub use partially_directed_acyclic_graph::LoadError;
pub use partially_directed_acyclic_graph::PDAG;
