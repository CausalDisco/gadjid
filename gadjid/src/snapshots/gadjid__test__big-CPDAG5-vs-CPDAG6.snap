---
source: src/lib.rs
expression: "test(\"10005.CPDAG-100\", \"10006.CPDAG-100\")"
---
g_true: 10005.CPDAG-100
g_guess: 10006.CPDAG-100
ts:
  - 83
  - 15
  - 61
  - 7
  - 92
ys:
  - 15
  - 50
  - 79
  - 52
  - 33
zs:
  - - 5
    - 6
    - 7
    - 12
    - 22
    - 23
    - 25
    - 26
    - 27
    - 30
    - 36
    - 47
    - 48
    - 59
    - 67
    - 70
    - 72
    - 75
    - 81
  - - 1
    - 10
    - 11
    - 14
  - - 1
    - 20
    - 23
    - 24
    - 34
    - 38
    - 40
    - 41
    - 42
    - 45
    - 47
    - 52
    - 53
    - 54
    - 58
  - []
  - - 2
    - 4
    - 5
    - 14
    - 25
    - 26
    - 28
    - 29
    - 30
    - 31
    - 34
    - 36
    - 44
    - 45
    - 53
    - 60
    - 69
    - 70
    - 74
    - 76
    - 87
nams:
  - []
  - []
  - []
  - []
  - []
nvas:
  - - 5
    - 6
    - 7
    - 12
    - 22
    - 23
    - 25
    - 26
    - 27
    - 30
    - 36
    - 47
    - 48
    - 59
    - 67
    - 70
    - 72
    - 75
    - 81
  - - 1
    - 10
    - 11
    - 14
  - - 1
    - 20
    - 23
    - 24
    - 34
    - 38
    - 40
    - 41
    - 42
    - 45
    - 47
    - 52
    - 53
    - 54
    - 58
  - []
  - - 2
    - 4
    - 5
    - 14
    - 25
    - 26
    - 28
    - 29
    - 30
    - 31
    - 34
    - 36
    - 44
    - 45
    - 53
    - 60
    - 69
    - 70
    - 74
    - 76
    - 87
pa_true_1st_T:
  - 5
  - 6
  - 7
  - 12
  - 22
  - 23
  - 25
  - 26
  - 27
  - 30
  - 36
  - 47
  - 48
  - 59
  - 67
  - 70
  - 72
  - 75
  - 81
an_true_1st_T:
  - 0
  - 1
  - 2
  - 3
  - 4
  - 5
  - 6
  - 7
  - 8
  - 9
  - 10
  - 11
  - 12
  - 13
  - 14
  - 15
  - 16
  - 17
  - 18
  - 19
  - 20
  - 21
  - 22
  - 23
  - 24
  - 25
  - 26
  - 27
  - 28
  - 29
  - 30
  - 31
  - 32
  - 33
  - 34
  - 35
  - 36
  - 37
  - 38
  - 39
  - 40
  - 41
  - 42
  - 43
  - 44
  - 45
  - 46
  - 47
  - 48
  - 49
  - 50
  - 51
  - 52
  - 53
  - 54
  - 55
  - 56
  - 57
  - 58
  - 59
  - 61
  - 62
  - 63
  - 64
  - 65
  - 67
  - 68
  - 69
  - 70
  - 72
  - 75
  - 81
  - 83
ch_true_1st_T:
  - 97
de_true_1st_T:
  - 83
  - 97
poss_de_true_1st_T:
  - 83
  - 97
prop_an_true_1st_T_and_1st_Y:
  - 1
  - 2
  - 6
  - 7
  - 8
  - 9
  - 10
  - 11
  - 14
  - 15
shd:
  - 0.006262626262626263
  - 31
oset_aid:
  - 0.02666666666666667
  - 264
parent_aid:
  - 0.04181818181818182
  - 414
ancestor_aid:
  - 0.02494949494949495
  - 247
